approval_polls/staticfiles/create.js

```
@@ -15,16 +15,47 @@ $(function () {
+    pollOptions.container.on("paste", "input[type=text]", function (e) {
+      // Get the pasted content
+      const pastedData = e.originalEvent.clipboardData.getData("text");
+
+      // Check if the pasted content contains newlines
+      if (pastedData.includes("\n")) {
+        e.preventDefault(); // Prevent default paste
+
+        // Split the pasted content into lines
+        const lines = pastedData
+          .split("\n")
+          .map((line) => line.replace(/^[-•*⁃◦▪︎●■]+/, "").trim())
```

Biome (1.9.4)

[error] A character class cannot match a character and a combining character.

A character and a combining character forms a new character. Replace the character class with an alternation.

(lint/suspicious/noMisleadingCharacterClass)